axelor-account/src/main/java/com/axelor/apps/account/web/AccountingReportController.java

```diff
@@ -228,4 +239,19 @@ public void showMoveExported(ActionRequest request, ActionResponse response) {
 
     response.setView(actionViewBuilder.map());
   }
+
+  public void setAccountTypeForAccountReport(ActionRequest request, ActionResponse response) {
+
+    AccountingReport accountingReport = request.getContext().asType(AccountingReport.class);
+    if (accountingReport.getTypeSelect() == 1000) {
```

Comment:
dont' use static values but constant from AccountingReportRepository for selection